app/assets/javascripts/map.js

```diff
@@ -0,0 +1,56 @@
+$(window).load(function() {
+  loadScript();
+});
+
+var latLng, map;
+function initMap(restaurantLoc) {
```

Comment:
'initMap' is defined but never used.<br>'restaurantLoc' is defined but never used.